detd/systemconf.py

```diff
@@ -218,13 +219,13 @@ def __init__(self):
         pass
 
 
-    def setup(self, interface, mapping, scheduler, base_time):
+    def setup(self, interface, mapping, scheduler, base_time, hints):
         tc = CommandTc()
 
-        if interface.device.supports_qbv():
+        if (hints.tx_selection == TxCapability.Qbv or hints.tx_selection == TxCapability.Qbu) and hints.tx_selection_offload == True:
```

Comment:
Please use the following:

If the hints supply EST:

If offload is true: taprio_offload
elif ltc is true: taprio_txassist, but raise an unimplemented error for the time being
else: taprio_software

elif the hints supply STRICT_PRIORITY:
throw unimplemented for the time being

else throw error (should not happen, we can also assert as the check should be performed when the hints are provided)